File: Week 6 Python/mario.py
from cs50 import get_int

height = get_int('Height: ')

# input validation
while height < 1 or height > 8:
    height = get_int('Height: ')

# print pyramid
for row in range(1, height+1):
    print(' ' * (height - row), end="")
    print('#' * row, end="  ")
    print('#' * row)

File: Week 1 C/mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height = get_int("Height: ");

    while (!(height >= 1 && height <= 8))
    {
        height = get_int("Height: ");
    }

    int space = height - 1;

    for (int row = 1; row <= height; row++)
    {
        for (int s = 0; s < space; s++)
        {
            printf(" ");
        }

        for (int c = 1; c <= row; c++)
        {
            printf("#");
        }

        printf("  ");

        for (int c = 1; c <= row; c++)
        {
            printf("#");
        }

        space--;
        printf("\n");
    }
}

// Height: 4
//    #  #
//   ##  ##
//  ###  ###
// ####  ####